playpen/all_26.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash -e
+
+
+# You need the .bashrc that the Vagrant or dev-setup.sh environment creates. You can get it from
+# playpen/ansible/roles/dev/files/bashrc if you aren't using one of those envs.
+. ~/.bashrc
+
+# Switch back to pulp-2.6
+pstop
+
+pushd ~/devel
+for r in {pulp_deb,pulp_docker,pulp_openstack,pulp_ostree,pulp_puppet,pulp_python,pulp_rpm,pulp}; do
```

Comment:
Brace expansion is a feature only available to certain shells, like bash and csh. How about either dropping it:

```sh
for r in pulp_deb pulp_docker pulp_openstack pulp_ostree ... ; do
```

Or taking full advantage of it:

```sh
for r in pulp pulp_{deb,docker,openstack,ostree,puppet,python,rpm}; do
```